ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_splitting_arrays.py

```diff
@@ -0,0 +1,30 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers import array_indices_axis
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# split
+@handle_frontend_test(
+    fn_tree = "numpy.split",
+    indices_or_sections_ary_axis=array_indices_axis

```

Comment:
You're getting an error because you aren't passing any arguments to this function, please have a look at its implementation in other tests, as well as the function itself. You are not even calling the function properly !